tests/applications/sel_phase_optimizer/test_sel_phase_linac.py

```diff
@@ -28,13 +28,13 @@ def cavity() -> SELCavity:
 
 
 def test_sel_poff_pv(cavity):
```

Comment:
Can you change the function name too sorry lolll

Reply:
done.